test/conformance/configuration_test.go

```diff
@@ -41,7 +40,8 @@ func TestUpdateConfigurationMetadata(t *testing.T) {
 	test.CleanupOnInterrupt(func() { tearDown(clients, names) }, logger)
 
 	logger.Infof("Creating new configuration %s", names.Config)
-	if _, err := test.CreateConfiguration(logger, clients, names, test.ImagePath(pizzaPlanet1), &test.Options{}); err != nil {
+	_, err := test.CreateConfiguration(logger, clients, names, test.ImagePath(pizzaPlanet1), &test.Options{})
```

Comment:
I think previous version was better?

Reply:
This is the first place err is defined. Below it has a member variable and err being returned which prohibits using := so I figured going back to this was easier than creating a temporary variable in the statement below.